GeneratorInterface/LHEInterface/src/HighFive/CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+##Version 2.0 - 2018/07/19
```

Comment:
this is a copy of someone else's code
we show just make it an external package and let them maintain it.